cmd/neofs-adm/internal/modules/morph/netmap_candidates.go

```diff
@@ -0,0 +1,46 @@
+package morph
+
+import (
+	"fmt"
+
+	"github.com/nspcc-dev/neo-go/pkg/rpcclient/invoker"
+	"github.com/nspcc-dev/neofs-node/cmd/internal/cmdprinter"
+	"github.com/nspcc-dev/neofs-node/pkg/morph/client/netmap"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func listNetmapCandidatesNodes(cmd *cobra.Command, _ []string) error {
+	c, err := getN3Client(viper.GetViper())
+	if err != nil {
+		return err
+	}
+
+	inv := invoker.New(c, nil)
+
+	nnsCs, err := c.GetContractStateByID(1)
+	if err != nil {
+		return fmt.Errorf("can't get NNS contract info: %w", err)
+	}
+
+	nmHash, err := nnsResolveHash(inv, nnsCs.Hash, netmapContract+".neofs")
+	if err != nil {
+		return fmt.Errorf("can't get netmap contract hash: %w", err)
+	}
+
+	res, err := inv.Call(nmHash, "netmapCandidates")
+	if err != nil {
+		return fmt.Errorf("can't fetch list of network config keys from the netmap contract: %w", err)
+	}
+
+	nm, err := netmap.DecodeNetMap(res.Stack)
```

Comment:
@roman-khimov, dont we need to check `HALT`?

Reply:
Likely so. Unfortunately, there is no appropriate `unwrap`. Contract wrapper can be tried, of course, but let's leave it for now.

Reply:
added